lib/main.ts

```diff
@@ -60,14 +63,29 @@ class AchYouDataProviderEngine
     return true;
   }
 
+  /**
+   * @param selector string userId of new user
```

Comment:
It appears this selector can only be 'NEWUSER'. Is it intended to do anything if you provide a selector with anything other than 'NEWUSER'? I would imagine that would edit an existing user, but it doesn't seem to be doing that right now. However you want to write it is fine - just a bit confused between the parameter description and its functionality.